libs/klv/src/test/java/org/codice/alliance/libs/klv/TestLatitudeLongitudeHandler.java

```diff
@@ -32,6 +35,8 @@
 
     private static final String LON = "lon";
 
+    private static final double EPSILON = 0.01;
```

Comment:
I could make the epsilon smaller. It's just the fudge factor for comparing floating point numbers.